src/app/profiles/store/selectors/index.ts

```diff
@@ -48,6 +49,20 @@ export const getSimilarContributors = createSelector(
     )
 );
 
+export const getFamousContributors = createSelector(
```

Comment:
```
export const getFamousContributors = createSelector(
  getStatefulContributors,
  statefulContributors => famousContributors.map(
    name => statefulContributors.find(contributor => name === contributor.name)
  )
);
```

OR 

```
export const getFamousContributors = createSelector(
  getStatefulContributors,
  statefulContributors => famousContributors.map(
    id => statefulContributors.find(contributor => id === contributor.id)
  )
);
```

if we change to storing ids …

Reply:
>  statefulContributors => famousContributors.map(
>     name => statefulContributors.find(contributor => name === contributor.name)
>   )

It's not working like that because `getStatefulContributors` is fired 2 times : 
- first time to get user page
- second time to get all statefull contributors
i don't know why but it same with getSimilarContributors selector

So if we use map, the first time give an array with empty value > it break the component without lot of test to valid. 
If we use foreach and push an init array empty there is no error and no need more test.